Automattic/wp-calypso
# update hero flow illustrations

#### Changes proposed in this Pull Request

Replace the hero flow illustrations with new ones from here p1637183547475700-slack-C029SB8JT8S
Also update the padding to 64px as shown in the figma files EuqfSnWpfYx8fgiBlVmbuA-fi-3817%3A1643

#### Testing instructions
Click through the hero flow and observe new images
starting from:
`start?flags=signup/hero-flow` or 
`start/setup-site/intent?siteSlug=YOUR_TEST_SITE.wordpress.com&flags=signup/hero-flow`
